src/+otp/+arenstorf/f.m

```diff
@@ -0,0 +1,8 @@
+function dy = f(~, y, m1 ,m2)
+
+    D1= ((y(1)+m1)^2+y(2)^2)^(3/2);
+    D2= ((y(1)-m2)^2+y(2)^2)^(3/2);

```

Comment:
here for readability I would like spaces between the pluses and minuses. in this case it would really help. 

Oh, and a space before the equals sign as well.